src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingRequestToCsvMapper.java

```diff
@@ -13,7 +13,10 @@ public class GetHearingRequestToCsvMapper {
 
     public HearingRequestForCsv toHearingRequestForCsv(CaseHearingRequestEntity requestEntity) {
         HearingRequestForCsv hearingRequestForCsv = new HearingRequestForCsv();
-        hearingRequestForCsv.setCaseReference(requestEntity.getCaseReference());
+        //numbers greater than 15 digits results in last number always being a 0
+        //https://learn.microsoft.com/en-US/office/troubleshoot/excel/last-digits-changed-to-zeros
+        hearingRequestForCsv.setCaseReference(String.format(requestEntity.getCaseReference() + "\t"));
```

Comment:
the link says add a quotation mark but this adds a tab. Presuming the same result.

Reply:
Same result - client preferred this option instead, incase we copy and paste the value to search